pom.xml

```diff
@@ -24,10 +24,10 @@
         <groupId>org.kill-bill.billing</groupId>
         <version>0.85</version>
     </parent>
-    <groupId>org.kill-bill.billing.plugin.java</groupId>
-    <artifactId>killbill-email-notifications-plugin</artifactId>
-    <name>Kill Bill OSGI Email Notifications Plugin</name>
-    <version>0.2.1</version>
+    <groupId>com.womply.billing</groupId>
+    <artifactId>womply-killbill-email-notifications-plugin</artifactId>
+    <name>Womply fork of Kill Bill OSGI Email Notifications Plugin</name>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
yeah wasn't sure if we want to regroup/artifact this, or just set version to 0.2.1-WOMPLY-<build_number> ... i'll tweak that in my subtask